timepiece/entries/views.py

```diff
@@ -1022,3 +1022,20 @@ def toggle_pause_entry(request):
         return JsonResponse(active_entry.to_json())
     else:
         return HttpResponse('')
+
+@login_required
+def get_verification_information(request):
+    pay_period_entries = Entry.objects.filter()
+
+    (period_start, period_end) = utils.get_last_bimonthly_dates()
+    entries = request.user.timepiece_entries.filter(
+        Q(status=Entry.UNVERIFIED),
+        # start_time__gte=period_start,
+        # end_time__lt=period_end
+        start_time__gte=period_start
+    )
+
+    return JsonResponse({
+        'verified': bool(entries),
```

Comment:
I don't really like this variable name.  If there are unverified entries, then this will return `{'verified': True}` which seems misleading.  Could we change it?